web/share/js/kvm/mouse.js

```diff
@@ -243,37 +249,43 @@ export function Mouse(__getGeometry, __recordWsEvent) {
 			});
 		}
 	};
-
 	var __streamWheelHandler = function(event) {
 		// https://learn.javascript.ru/mousewheel
 		// https://stackoverflow.com/a/24595588
 
 		event.preventDefault();
-
+		//set default rate of -5, but allow localStorage hid.mouse.scroll_rate value to be used.
+		var rate=-5;
+		try{
+			rate=-window.localStorage.getItem("hid.mouse.scroll_rate");
+		} catch (ReferenceError){
+			// no rate was d
+		}
```

Comment:
Create class variable like `__relative_sens`, update it in `__updateScrollRate()` (see `__updateRelativeSens()`) and use it here. This code is executed too often and it is better to do without accessing the browser storage.

Reply:
Fixed